client/lib/follow-list/test/index.js

```diff
@@ -1,14 +1,12 @@
+/** @jest-environment jsdom */
+jest.mock( 'lib/wp', () => require( './lib/wp' ) );
```

Comment:
Any reason not to move `./lib/wp.js` to this file? The difference between `lib/wp` and `./lib/wp` is subtle and it took me some time to figure out the code came from the local test directory.